apps/docs/src/development/packages/nuxt.md

```diff
@@ -45,12 +45,32 @@ If your Nuxt project uses both modules, onyx will automatically detect it and us
 
 ### Setup
 
-The setup is automated, @sit-onyx/nuxt will check if @nuxtjs/i18n is registered as well and register all onyx translations and add a plugin to handle the translations for onyx using vue-i18n.
+To merge the locales provided by onyx with your own, you need to define a mapping.
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  modules: ["@sit-onyx/nuxt", "@nuxtjs/i18n"],
+  i18n: {
+    defaultLocale: "en_US",
+    langDir: "./i18n",
+    locales: [
+      { code: "en_US", file: "en-US.json", name: "English (US)" },
+      { code: "de", file: "de-DE.json", name: "Deutsch" },
+    ],
+  },
+  onyx: {
+    i18n: {
+      registerLocales: {
+        en_US: "en-US",
```

Comment:
Is the mapping also needed when the key and value is the same? If yes, we should maybe also add an example for this here so people might not think they only need it when they are different